packages/channel-client/src/fake-channel-client.ts

```diff
@@ -12,21 +12,19 @@ import {
   Allocation,
   PushMessageResult
 } from './types';
-import {sleep} from './utils';
 
-const FAKE_DELAY = 100; // ms
+export const CHANNEL_ID = '0x1234';
```

Comment:
Doesn't this mean we can only create one channel with this fake client? This should probably be like the hash of the arguments or something like that.